### PR TITLE
Changed the default stdouts

### LINKED ISSUE #741: Change STDOUT on server start
When starting MindsDB if the user provides -v argument it should print all the logs. Otherwise, just print the message bellow.

```Versions:
 - lightwood 0.41.0
 - MindsDB_native 2.12.0
 - MindsDB 2.8.1

MindsDB storage directory: /Users/ricram2/opt/anaconda3/envs/mindsdb/lib/python3.8/site-packages/mindsdb_native/mindsdb_storage/2_12_0 does not exist and could not be created, trying another directory
Using configuration file: /Users/ricram2/opt/anaconda3/envs/mindsdb/lib/python3.8/site-packages/etc/config.json

Starting Mindsdb http API ... OK 
Started Mindsdb mysql API ... OK

You can now connect to MindsDB Server on any Mysql client as:

	 tcp://127.0.0.1:47335

Or You can work on MindsDB in your browser at: 

	http://localhost:3000

Happy Predicting!  More information at: https://docs.mindsdb.com

### PR DESCRIPTION
**Pull request by pushp1997**
Fixes #741 

## Please describe what changes you made, in as much detail as possible
  - Have changed the default STDOUT message and if verbose is not given a NOTE is displayed stating that the verbose is off:
SS without -v

  - If --verbose is not given log level is set to "ERROR" else it is set to "INFO" (NOTE: This was already done, yo may find it in __min__.py file, line 148):
image

mindsdb

### CODE REVIEW

**Comment by pushp1997** [comment]
Please let me know if any other change is required.

**Comment by ZoranPandovski** [comment]
recheckcla

**Comment by mindsdbadmin** [comment]
**CLA Assistant Lite bot:** <br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/mindsdb/mindsdb_native/blob/stable/assets/contributions-agreement/individual-contributor.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.<br/> 
   - - -
   ***I have read the CLA Document and I hereby sign the CLA***
   - - - 
   <sub>You can retrigger this bot by commenting **recheckcla** in this Pull Request</sub>

**Comment by George3d6** [comment]
@pushp1997 can you please sign the CLA ? We need it signed for external contributors since the code of Mindsdb is licensed under GPL

**Comment by pushp1997** [comment]
I have read the CLA Document and I hereby sign the CLA

File: mindsdb/__main__.py [hunk 1]
```diff
@@ -164,9 +169,17 @@ def close_api_gracefully(p_arr):
                 in_use = True
             if in_use and api['started'] != in_use:
                 api['started'] = in_use
-                print(f"{api['name']} API: started on {api['port']}")
+                if api['name'] == 'mysql':
+                    print("You can now connect to MindsDB Server on any Mysql client as:")
+                    print(f"\ttcp://127.0.0.1:{api['port']}")
+                else: # api['name'] == http
```
Comment (George3d6) [inline mindsdb/__main__.py:175]: Should be an `if` here since both can start at once

File: mindsdb/__main__.py [hunk 2]
```diff
@@ -149,6 +144,16 @@ def close_api_gracefully(p_arr):
...
 
+    os.environ['MINDSDB_STORAGE_PATH'] = config.paths['predictors']
+    if args.verbose is True:
+        os.environ['DEFAULT_LOG_LEVEL'] = 'INFO'
+        os.environ['LIGHTWOOD_LOG_LEVEL'] = 'INFO'
+    else:
+        os.environ['DEFAULT_LOG_LEVEL'] = 'ERROR'
+        os.environ['LIGHTWOOD_LOG_LEVEL'] = 'ERROR'
+        print("\nNOTE: Verbose is OFF [see --help for options]")
+    print('\n' + '='*7 + '\n') # print divider
```
Comment (George3d6) [inline mindsdb/__main__.py:155]: can you move these back to their original position ? If not, why did you move these down ?

**Review by George3d6 — CHANGES_REQUESTED** [review]

**Comment by George3d6** [comment]
@pushp1997 will close this for now, let me know if you want to start back up on it and we can reopen

**Pull request closed without merging**